config/environments/development.rb

```diff
@@ -61,4 +61,6 @@
   config.rdf_uri = 'https://researchdatabase.minneapolisfed.org'
 
   config.bag_prefix = 'mpls_fed_research'
+  # Whitelist default Docker IP and Docker IP used by project developers
+  config.web_console.whitelisted_ips = ["127.0.0.1", "::1", "127.0.0.0/8", "::1", "10.160.0.0/16", "::1", "172.17. 0.0/16", "::1"]
```

Comment:
I don't understand why the IPv6 "::1" entry shows up in this list four times.  Is it redundant and could we remove the extra copies OR could we put a comment (probably in the code rather than the PR) about why they need to be there - otherwise I'd be afraid that some well intentioned future dev would go through and remove them as being duplicative.

Reply:
Turns out it works fine without, since there were 2 in the original array, I thought they were in pairs, but nope!